client/src/pages/game/GamePrompt.js

```diff
@@ -14,7 +29,7 @@ function GamePrompt({ gameState, player }) {
               <TimerIcon fontSize="large" />
             </Grid>
             <Grid item>
-              <Typography variant="h6">45s left</Typography>
+              <Typography variant="h6">{gameState.winner ? 0 : countDown}s left</Typography>
```

Comment:
Just curious - why not just have it set as just countdown? I don't see the need to just show a 0 on the timer when the game is over

Reply:
Oh, it's because It feels odd to me that the timer is running after the game ends